conf/modules.config

```diff
@@ -154,6 +157,7 @@ process {
         time = { check_max( 16.h  * task.attempt, 'time' ) }
         ext.args = {
             [
+                "--genomicsdb-vcf-buffer-size 16384000",
```

Comment:
how did you get this number?

Reply:
A suggestion from a developer on a forum post [here](https://sites.google.com/a/broadinstitute.org/legacy-gatk-forum-discussions/2020-01-07-2019-07-10/24420-GenomicsDBImport-error-File2TileDBBinaryException) (the comment from 2019-09-10)